ferveo-python/test/test_ferveo.py

```diff
@@ -113,30 +114,30 @@ def scenario_for_variant(variant, shares_num, threshold, shares_to_use):
 
 
 def test_simple_tdec_has_enough_messages():
-    scenario_for_variant("simple", shares_num=4, threshold=3, shares_to_use=3)
+    scenario_for_variant(FerveoVariant.simple, shares_num=4, threshold=3, shares_to_use=3)
 
 
 def test_simple_tdec_doesnt_have_enough_messages():
-    scenario_for_variant("simple", shares_num=4, threshold=3, shares_to_use=2)
+    scenario_for_variant(FerveoVariant.simple, shares_num=4, threshold=3, shares_to_use=2)
 
 
 def test_precomputed_tdec_has_enough_messages():
-    scenario_for_variant("precomputed", shares_num=4, threshold=4, shares_to_use=4)
+    scenario_for_variant(FerveoVariant.precomputed, shares_num=4, threshold=4, shares_to_use=4)
 
 
 def test_precomputed_tdec_doesnt_have_enough_messages():
-    scenario_for_variant("precomputed", shares_num=4, threshold=4, shares_to_use=3)
+    scenario_for_variant(FerveoVariant.precomputed, shares_num=4, threshold=4, shares_to_use=3)
 
 
 PARAMS = [
-    (1, 'simple'),
-    (4, 'simple'),
-    (8, 'simple'),
-    (32, 'simple'),
-    (1, 'precomputed'),
-    (4, 'precomputed'),
-    (8, 'precomputed'),
-    (32, 'precomputed'),
+    (1, FerveoVariant.simple),
+    (4, FerveoVariant.simple),
+    (8, FerveoVariant.simple),
+    (32, FerveoVariant.simple),
+    (1, FerveoVariant.precomputed),
+    (4, FerveoVariant.precomputed),
+    (8, FerveoVariant.precomputed),
+    (32, FerveoVariant.precomputed),
```

Comment:
Let's introduce some odd numbers to this parametrization, after it's supported (now?).

Reply:
:heavy_check_mark: